apiserver/facades/client/cloud/register.go

```diff
@@ -55,7 +55,7 @@ func newFacadeV7(context facade.ModelContext) (*CloudAPI, error) {
 		}
 
 		return service.CredentialValidationContext{
-			ControllerUUID: m.ControllerUUID(),
```

Comment:
Not to do with your change, but we shouldn't even be getting this model (m) out of legacy state.